bors.toml

```
@@ -0,0 +1,8 @@
+required_approvals = 1
+status = [
```

I'd maybe

```suggestion
status = [
```

and rely on bors mechanism for approvals. 

I think that bors's way (r+ comments) is better. Sepcifically, reviewer, after they are satisfied with the code, leaves the `bors r+` comment, and that is considered an approval that triggers the merge, regardless of the presence of GH-level approval. This is better because: 

* It removes the ambiguity for "who presses the merge button". Reviewer does that, by explicitly leaving the `r+` comment. So, ultimately it's the reviewer who is on the hook to make  sure that the code we merge is good, which I think is a good psychological split of responsibilities. This also dovetails with idea I've heard from @jorangreef  that we deliberately have only a single reviewer to avoid bystander problem
* It removes ambiguity of what exactly is approved: `r+` always applies to the current commit hash.
* It makes "LGTM, but please this one small nit before merging" explicit: there's `bors d+` command which means exactly "the PR author can now approve their own change".

---

But also, I'd be totally up for merging this as is, and tweaking config later!

---

I like how that sounds - I went with `required_approvals` just to keep change down to a minimum. Deferring to @sentientwaffle here.

---

Agreed, that sounds more convenient — being able to say "PR author can approve their own change" is perfect.